### DISCUSSION TITLE
Changing the content in the prompt from unanswered "required" question in surveytext plugin?

### DISCUSSION THREAD

**Discussion by anjiecao** [jspsych/jsPsych#878] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/30306411/87685438-650f4780-c751-11ea-8943-073de06176ee.png)
Is there a way to change "Please fill out this field" into something else? I am trying to adapt my experiment into another language. But I can't find where this line is specified in js file. Any input would be appreciated!

**Comment by vijaymarupudi** [comment, marked as answer]
That comes from the browser, and will be in the language the participant sets their browser to!

**Reply by anjiecao** [reply]
ah great! thanks!
